sigstore-java/src/main/java/dev/sigstore/fulcio/client/FulcioClient.java

```diff
@@ -45,19 +43,19 @@
 public class FulcioClient {
 
   private final HttpParams httpParams;
-  private final CertificateAuthority certificateAuthority;
+  private final URI uri;
 
   public static Builder builder() {
     return new Builder();
   }
 
-  private FulcioClient(HttpParams httpParams, CertificateAuthority certificateAuthority) {
-    this.certificateAuthority = certificateAuthority;
+  private FulcioClient(HttpParams httpParams, URI uri) {
+    this.uri = uri;
     this.httpParams = httpParams;
   }
 
   public static class Builder {
-    private CertificateAuthority certificateAuthority;
+    private URI uri = URI.create("https://fulcio.sigstore.dev");
```

Comment:
How will staging be handled? Or is this just a default?

Reply:
This is just a default. The builder allows this to be configurable.